.codecov.yml

```diff
@@ -5,6 +5,7 @@ codecov:
     require_ci_to_pass: yes
 
 ignore:
+  - "policy/policy.pb.go"
```

Comment:
Should this be generalized to any `*.pb.go` file?

Reply:
I agree. Thanks